ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/RpcApis.java

```diff
@@ -57,9 +56,8 @@
       return Optional.of(PRIV);
     } else if (name.equals(TX_POOL.getCliValue())) {
       return Optional.of(TX_POOL);
-      // Disable TRACE functionality while under development
-      //    } else if (name.equals(TRACE.getCliValue())) {
-      //      return Optional.of(TRACE);
+    } else if (name.equals(TRACE.getCliValue())) {
+      return Optional.of(TRACE);
```

Comment:
Looks like this change should be undone (this `TRACE` option should be commented out) to leave the tracing API's disabled.